sdk/storage/azure-storage-blob/swagger/README.md

```diff
@@ -16,7 +16,7 @@ autorest --v3 --python
 
 ### Settings
 ``` yaml
-input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/specification/storage/data-plane/Microsoft.BlobStorage/preview/2021-04-10/blob.json
+input-file: https://github.com/seanmcc-msft/azure-rest-api-specs/blob/211765a5a0126610efde8e57b88a2cfa53c35275/specification/storage/data-plane/Microsoft.BlobStorage/preview/2021-08-06/blob.json
```

Comment:
Now that Sean has merged the STG83 Swagger to main we can reference main again here:
`https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/specification/storage/data-plane/Microsoft.BlobStorage/preview/2021-08-06/blob.json`.

Maybe just try generating from this link to make sure it works.

Reply:
I have gone ahead and done that, seems to have worked 😄